### ISSUE TITLE
[TypeScript] `HomeyAPIxxx.ManagerDevices.Device` misses properties (`capabilities`, `capabilitiesObj`...)

### ISSUE THREAD

**Issue by OlivierZal** [athombv/web-api-issues#29]
Fact that `HomeyAPIV2.ManagerDevices.Device` is typed as `object` raises errors for such lines of code:
```
const capability: string = 'target_temperature'
const value: any = device.capabilitiesObj[capability]?.value
```

```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'. ​typescript(7053)
  No index signature with a parameter of type 'string' was found on type '{}'.
```

or

```
const value: any = device.capabilitiesObj.target_temperature?.value
```

```
Property 'target_temperature' does not exist on type 'object'. ​typescript(2339)
```

**Comment by jeroenwienk** [comment]
I think you mean the problem is that capabilitiesObj is types as any?

**Comment by OlivierZal** [comment]
Should be rather `Record<string, any>`

**Comment by OlivierZal** [comment]
Yes @jeroenwienk, now it's `any` but before `homey-api@3.0.0` it was `object`.

The problem is that the type `Device` is not complete, as you can see on the following screenshots:
```Property 'capabilities' does not exist on type 'Device'.ts(2339)```
```Property 'capabilitiesObj' does not exist on type 'Device'.ts(2339)```

<img width="2240" alt="Capture d’écran 2023-07-24 à 23 25 45" src="https://github.com/athombv/homey-web-api-issues/assets/88216225/0ac98985-eea3-42c6-a894-61e46b2b1d21">

<img width="2240" alt="Capture d’écran 2023-07-24 à 23 25 28" src="https://github.com/athombv/homey-web-api-issues/assets/88216225/d315e87e-dcb4-4026-8ed0-d6d754d0cb00">

**Comment by OlivierZal** [comment]
Obsolete, since types have been recently added.
